setup/github-bot/resources/github-bot-deploy-webhook.json

```diff
@@ -0,0 +1,13 @@
+{
+  "port": 9999,
+  "path": "/deploy-webhook",
+  "secret": "{{ envs.github_webhook_secret }}",
```

Comment:
On second thought, we should probably use a separate github secret for this webhook? Meaning not use the same secret as we've got on *actual* bot webhook...